pkg/server/sandbox_status_test.go

```diff
@@ -90,22 +90,17 @@ func getSandboxStatusTestData() (*metadata.SandboxMetadata, *runtime.PodSandboxS
 
 func TestToCRISandboxStatus(t *testing.T) {
```

Comment:
We can get rid of this test now. Previously we need this to test the behavior of the `Network` field, but now that field is removed.